spec.md

```diff
@@ -51,9 +51,12 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 
 The high-level components of the spec include:
 
-* An archival format for container images, consisting of an [image manifest](manifest.md), an [image index](image-index.md) (optional), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A [process of referencing container images by a cryptographic hash of their content](descriptor.md) (base OCI layer)
-* A format for [storing CAS blobs and references to them](image-layout.md) (optional OCI layer)
+* [Image Manifest](manifest.md) - the stored document describing an image's properties
+* [Image Index](image-index.md) - the stored document describing a collection of images redistributed as one
```

Comment:
Is an image index really a collection of images redistributed _as one_? (It sort of implies it's bundled together, is that necessarily the case?)